.github/workflows/push.yml

```
@@ -14,12 +14,17 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v3
 
+      # Setup Node.js
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
```

actionlint (1.7.4)

the runner of "actions/setup-node@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)